tests/tests.js

```diff
@@ -172,7 +172,7 @@ define(['jquery', 'title', 'archive', 'zimArchive', 'zimDirEntry', 'util', 'geom
                 equal(title._name, "Diego_Velázquez", "Name of the title is correct");
                 start();
             };
-            localEvopediaArchive.getTitleByName("Diego_Velázquez").then(callbackFunction);
+            localEvopediaArchive.getTitleByName("A/Diego_Velázquez").then(callbackFunction);
```

Comment:
If I do that, the unit tests seem to run fine, so I'll merge your pull request and fix that myself on master